dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/src/org/apache/cxf/binding/soap/interceptor/RPCOutInterceptor.java

```diff
@@ -0,0 +1,165 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.cxf.binding.soap.interceptor;
+import java.util.List;
+import java.util.logging.Logger;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamWriter;
+import javax.xml.stream.events.XMLEvent;
+
+import org.apache.cxf.binding.soap.wsdl.extensions.SoapBody;
+import org.apache.cxf.common.logging.LogUtils;
+import org.apache.cxf.common.util.StringUtils;
+import org.apache.cxf.helpers.NSStack;
+import org.apache.cxf.interceptor.AbstractOutDatabindingInterceptor;
+import org.apache.cxf.interceptor.Fault;
+import org.apache.cxf.message.Message;
+import org.apache.cxf.message.MessageContentsList;
+import org.apache.cxf.phase.Phase;
+import org.apache.cxf.service.model.BindingOperationInfo;
+import org.apache.cxf.service.model.MessagePartInfo;
+import org.apache.cxf.staxutils.CachingXmlEventWriter;
+import org.apache.cxf.staxutils.StaxUtils;
+
+public class RPCOutInterceptor extends AbstractOutDatabindingInterceptor {
+    private static final Logger LOG = LogUtils.getL7dLogger(RPCOutInterceptor.class);
+
+    public RPCOutInterceptor() {
+        super(Phase.MARSHAL);
+    }
+
+
+    public void handleMessage(Message message) {
+        XMLStreamWriter origXmlWriter = null;
+        try {
+            NSStack nsStack = new NSStack();
+            nsStack.push();
+
+            BindingOperationInfo operation = message.getExchange().getBindingOperationInfo();
+
+            assert operation.getName() != null;
+
+            XMLStreamWriter xmlWriter = getXMLStreamWriter(message);
+            CachingXmlEventWriter cache = null;
+            // need to cache the events in case validation fails or buffering is enabled
+            if (shouldBuffer(message)) {
+                origXmlWriter = xmlWriter;
+                cache = new CachingXmlEventWriter();
+                try {
+                    cache.setNamespaceContext(xmlWriter.getNamespaceContext());
+                } catch (XMLStreamException e) {
+                    //ignorable, will just get extra namespace decls
+                }
+                message.setContent(XMLStreamWriter.class, cache);
+                xmlWriter = cache;
+            }
+
+            List<MessagePartInfo> parts = null;
+
+            boolean output = false;
+            if (!isRequestor(message)) {
+/*              Liberty change: if block is removed
+                if (operation.getOutput() == null) {
+                    return;
+                } */
+                parts = operation.getOutput().getMessageParts();
+                output = true;
+            } else {
+                parts = operation.getInput().getMessageParts();
+                output = false;
+            }
```

Comment:
This is the same block currently in CXF: 
```            boolean output = false;
            if (!isRequestor(message)) {
                parts = operation.getOutput().getMessageParts();
                output = true;
            } else {
                parts = operation.getInput().getMessageParts();
                output = false;
            }```

You can remove this change.

Reply:
This if block exists in CXF 3.4.1. You want me to uncomment this block right?               
                if (operation.getOutput() == null) {
                    return;
                }

Reply:
Remove the class